### DISCUSSION TITLE
[BUG] renderer为svg时，图例legend分页点击上一页无反应

### DISCUSSION THREAD

**Discussion by ufresh2013** [antvis/G2#5093] [Q&A] [answered]
🐛 Bug description
renderer为svg时，图例legend分页点击上一页无反应

📷 Step to reproduce
```
import { Column } from '@antv/g2plot';

fetch('https://gw.alipayobjects.com/os/antfincdn/P14mCvkybz/large-datra.json')
  .then((data) => data.json())
  .then((data) => {
    const plot = new Column('container', {
      data,
      xField: 'product_box',
      renderer:'svg',
      yField: 'value',
      seriesField: 'province',
      isGroup: 'true',
    });

    plot.render();
  });
```
🏞 Expected result
图例分页点击上一页能正常返回上一页，且缩放窗口大小，不会导致图例错位

🚑 Any additional
G2Plot Version: 2.4.25
Platform: Chrome 111.0.5563.65（正式版本） （64 位）

**Comment by hustcc** [comment, marked as answer]
G2Plot 2.4.25 底层使用的 G2 4.x 版本，对于 SVG 的支持确实不好，问题挺多，建议在 4.x 中不要使用 SVG。在新版本 5.0 中加强 SVG 的能力。

- ref https://github.com/antvis/G2Plot/discussions/3570
